clusters/hive/hypershift.yaml

```diff
@@ -113,3 +113,85 @@ subjects:
   - kind: ServiceAccount
     namespace: ci
     name: hypershift-workload
+---
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  name: hypershift-worker
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      machineset-role: hypershift
+  template:
+    metadata:
+      labels:
+        machineset-role: hypershift
+        machine.openshift.io/cluster-api-machine-role: worker
+        machine.openshift.io/cluster-api-machine-type: worker
+    spec:
+      metadata:
+        labels:
+          hypershift.openshift.io/control-plane: 'true'
```

Comment:
Does `Hypershift` set up node selector on the pods which we expect to run on the nodes from the machine set?

I guess that is what https://github.com/openshift/release/pull/43262/files#diff-6f6f9668e008a46382d5523177e1aeeb3f3d09f9c92baee608a2256b713552f6R63 is for.

Reply:
Correct, I changed the parameter of the workflow to apply the node selector